### PR TITLE
Add build/push docker info

### PR DESCRIPTION
**Pull request by wattli**

### CODE REVIEW

File: DOCKER-IAMGE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+If you want to make a local change and test some component, say istio-ca, you could follow below steps:
```
Comment (ldemailly) [inline DOCKER-IAMGE.md]: typo in file name - and please put this in the dev guide instead

File: DOCKER-IAMGE.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+Under istio/istio repo
+$ pwd .../istio.io/istio
```
Comment (rkpagadala) [inline DOCKER-IAMGE.md]: Change to 
```
$ pwd
.../istio.io/istio
```
And what laurent said

**Comment by wattli** [comment]
@ldemailly done.

File: DEV-GUIDE.md [hunk 3]
```diff
@@ -236,6 +237,38 @@ Push the containers to your registry:
...
+
+Under istio/istio repo
+
+```shell
+pwd
+```
+The path should be
+
+```shell
+.../istio.io/istio
```
Comment (ldemailly) [inline DEV-GUIDE.md]: .../src/istio.io/istio even but I expect this is covered in the rest already

File: DEV-GUIDE.md [hunk 4]
```diff
@@ -236,6 +237,38 @@ Push the containers to your registry:
...
+```
+
+Set up environment variables HUB and TAG by
+```shell
+export HUB=gcr.io/testing
```
Comment (ldemailly) [inline DEV-GUIDE.md]: maybe mention docker.io/yourdockerlogin ? gcr.io/testing isn't open to the world (I think/hope)

File: DEV-GUIDE.md (hunk 3 above)
Comment (wattli) [reply DEV-GUIDE.md]: Done

File: DEV-GUIDE.md (hunk 4 above)
Comment (wattli) [reply DEV-GUIDE.md]: Done

**Comment by wattli** [comment]
@ldemailly , the LGTM label is removed by the new commit, please take another look, thanks!

**Pull request merged**